chart/k8gb/templates/operator.yaml

```diff
@@ -27,7 +27,6 @@ spec:
       containers:
         - name: k8gb
           image: {{ .Values.k8gb.imageRepo }}:{{ .Values.k8gb.imageTag | default .Chart.AppVersion }}
-          imagePullPolicy: Always
```

Comment:
Can we please set `imagePullPolicy` explicitly to `IfNotPresent` ?  I know it is default but just to make it explicit